GOVERNANCE.md

```diff
@@ -0,0 +1,201 @@
+
+Automatic releases of the modules should be set up by the Core Team using the organization
+secret in the GitHub organization.
+
+Auxiliary projects can can be nominated to become a primary project by its Collaborators,
+by opening a [private team discussion on GitHub][].
+
+If a primary project is no longer widely depended upon, or have technical reasons to no
+longer stay a primary project, the Core Team can decide to convert it into an auxiliary
+project in discussion with its Collaborators.
+
+## Project nominations
+
+Projects that are widely depended upon within the ESLint ecosystem may be nominated to
+become ESLint-Community projects by anyone by opening a public issue on GitHub in the
+[eslint-community/governance][] repository.
+
+The project should fulfill a few criterias:
+
+The project should:
+
+* Preferably be: An ESLint plugin or formatter
+* Maybe be: A dependency used by an ESLint-Community project or the main ESLint repo
+* Not be: Primarily an ESLint config
+
+The project should also:
+
+* be widely depended upon throughout the community (eg. 3M downloads/week or similar)
+* have an OSI-approved license (preferably Apache 2.0, else MIT-style)
+* have proposed Collaborators or have possibility to be deemed a Primary project
+  by the Core Team
+
+If project currently has an active maintainer it should be transfered and then:
+
+* the current maintainer should be willing to transfer the project to ESLint-Community
+
+If it's an abandoned project that needs to be forked:
+
+* It shouldn't have received an update in a substantial amount of time (eg. last 6 months)
+  or it should present a major obstacle to the adoption of eg. a new major version of ESLint
+* A clear notice about the nomination should have been made to the project and its
+  maintainer at least 30 days before it can be accepted
+
+Projects should typically start out with a status of "auxiliary".
+
+## Collaborator nominations
+
+Individuals making significant and valuable contributions to a project may be
+a candidate to join the ESLint-Community organization and that team.
+
+An existing Collaborator needs to open a [private team discussion on GitHub][] and
+list the candidates they want to sponsor with a link to the user's contributions. For
+example:
+
+* Activities in the ESLint-Community organization
+  `[USERNAME](https://github.com/search?q=author:USERNAME+org:eslint-community)`
+
+Otherwise, a Contributor may self-apply if they believe they meet the above
+criteria by reaching out to a Core Team member privately with the links to their
+valuable contributions. That Lead Maintainer will then open a private team discussion on
+GitHub regarding that and reply back when a decision has been made.
+
+The consensus to grant a new candidate Collaborator status is reached when:
+
+- at least two of the Core Team members approve
+- at least half of the current collaborators in that team approve, rounded up
+  (and not counting Core Team members)
```

Comment:
Is this criteria going to be difficult to meet with people often becoming unreachable?

Reply:
Do you think the following explains how to deal with that case?

> Lazy means that silence is consent if an appropriate amount of time has passed (most of the time: at least a week) since the suggestion was made and no objections has been raised.

Reply:
It seems to me it would be a good approach to use.